deploy/jenkins-ci/deploy-kiali-for-system-tests-code-coverage.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+bash <(curl -L https://git.io/getLatestKialiOperator)
```

Comment:
Note that I just did this "curl" download simply to pick up the released version. This is simply running the stock [operator/deploy/deploy-kiali-operator.sh script](https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh). You can execute that script however you do it normally - if you are running this out of a git cloned repo, you could make this something like `../../operator/deploy/deploy-kiali-operator.sh` instead.